src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceDeclaration.vue

```diff
@@ -80,5 +82,18 @@ export default defineComponent({
       Rules,
     };
   },
+  computed: {
+    certificationType() {
+      let certificationType = "Certificate type not found";
+      if (this.wizardStore.wizardData.certificationTypes?.includes(CertificationType.ECE_ASSISTANT)) {
+        certificationType = "ECE Assistant certificate";
+      } else if (this.wizardStore.wizardData.certificationTypes?.includes(CertificationType.ONE_YEAR)) {
+        certificationType = "ECE One Year certificate";
+      } else if (this.wizardStore.wizardData.certificationTypes?.includes(CertificationType.FIVE_YEAR)) {
+        certificationType = "ECE Five Year certificate";
+      }
+      return certificationType;
+    },
+  },
```

Comment:
you did the work already here, so it's alright. I was thinking that we would only have a work reference for those with a five year certificate. 

So I just left it hardcoded initially.